lib/json_marshal/marshaller.rb

```diff
@@ -7,7 +7,7 @@ def self.dump(obj)
     end
 
     def self.load(attribute)
-      JSON.parse(attribute)
+      JSON.parse(attribute) unless attribute.nil?
```

Comment:
if attribute = nil JSON.parse(nil) throws an error, causing issues when marshaling/loading encrypted attrs (hashes, arrays, etc)

For example, some AsyncTransaction::Base records have metadata as nil, throwing a`TypeError (no implicit conversion of nil into String)`